catalog/spatial/registry/registry-schema-bindings/src/main/java/org/codice/ddf/registry/schemabindings/RegistryPackageWebConverter.java

```diff
@@ -0,0 +1,1760 @@
+        }
+    }
+
+    private static void populateRegistryObjectList(Map<String, Object> rolMap,
+            RegistryPackageType registryPackage) {
+        if (rolMap.isEmpty()) {
+            return;
+        }
+
+        RegistryObjectListType registryObjectList = RIM_FACTORY.createRegistryObjectListType();
+
+        if (rolMap.containsKey(EXTRINSIC_OBJECT_KEY)) {
+            populateROLWithExtrinsicObject((List<Map<String, Object>>) rolMap.get(
+                    EXTRINSIC_OBJECT_KEY), registryObjectList);
+
+        }
+
+        if (rolMap.containsKey(SERVICE_KEY)) {
+            populateROLWithService((List<Map<String, Object>>) rolMap.get(SERVICE_KEY),
+                    registryObjectList);
+        }
+
+        if (rolMap.containsKey(ORGANIZATION_KEY)) {
+            populateROLWithOrganization((List<Map<String, Object>>) rolMap.get(ORGANIZATION_KEY),
+                    registryObjectList);
+        }
+
+        if (rolMap.containsKey(PERSON_KEY)) {
+            populateROLWithPerson((List<Map<String, Object>>) rolMap.get(PERSON_KEY),
+                    registryObjectList);
+        }
+
+        if (rolMap.containsKey(ASSOCIATION_KEY)) {
+            populateROLWithAssociation((List<Map<String, Object>>) rolMap.get(ASSOCIATION_KEY),
+                    registryObjectList);
+        }
+
+        if (CollectionUtils.isNotEmpty(registryObjectList.getIdentifiable())) {
+            registryPackage.setRegistryObjectList(registryObjectList);
+        }
+
+    }
+
+    private static void putExtrinsicObject(ExtrinsicObjectType extrinsicObject,
+            Map<String, Object> registryObjectListMap) {
+        if (extrinsicObject == null) {
+            return;
+        }
+
+        Map<String, Object> extrinsicObjectMap = new HashMap<>();
+
+        putGeneralInfo(extrinsicObject, extrinsicObjectMap);
+
+        if (extrinsicObject.isSetContentVersionInfo()) {
+            extrinsicObjectMap.put(EXTRINSIC_OBJECT_CONTENT_VERSION,
+                    extrinsicObject.getContentVersionInfo()
+                            .getVersionName());
+        }
+
+        if (extrinsicObject.isSetIsOpaque()) {
+            extrinsicObjectMap.put(EXTRINSIC_OBJECT_OPAQUE, extrinsicObject.isIsOpaque());
+        }
+
+        if (extrinsicObject.isSetMimeType()) {
+            extrinsicObjectMap.put(EXTRINSIC_OBJECT_MIME_TYPE, extrinsicObject.getMimeType());
+        }
+
+        if (extrinsicObject.isSetExternalIdentifier()) {
+            putExternalIdentifier(extrinsicObject.getExternalIdentifier(), extrinsicObjectMap);
+        }
+
+        if (!extrinsicObjectMap.isEmpty()) {
+            registryObjectListMap.putIfAbsent(EXTRINSIC_OBJECT_KEY,
+                    new ArrayList<Map<String, Object>>());
+            ((List) registryObjectListMap.get(EXTRINSIC_OBJECT_KEY)).add(extrinsicObjectMap);
+        }
+    }
+
+    private static void populateROLWithExtrinsicObject(
```

Comment:
We should be consistent and always use `RegistryObjectList` in method and variable names instead of `rol` or `ROL`.